### ISSUE TITLE
When editing PDF changes made for 

### ISSUE THREAD

**Issue by Ephirion** [ONLYOFFICE/DesktopEditors#1687]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (RPM package)

### Version information

8.1.1.27 (rpm)

### Expected Behavior

When I change font color or any other text settings as well as text box shape settings (i.e. background, border, shadow, etc.) those changes should be saved and applied to the next created text box.

### Actual Behavior

Each new text box has default font and shape settings, which makes inserting text with consistent style extremely inconvenient and frustrating

### Reproduction Steps

1. Create a text box element, and change any font, font size, color, weight, and shape parameters.
2. Type text and create a new text box element.
3. Type new text and observe that all changes you've made reverted to default

**Comment by XDmitryK** [comment]
@Ephirion Hi
We have released version 8.2.0, can you update your version and check? If the problem still exists, can you record a short video about this problem?

**Comment by Ephirion** [comment]
No, the issue still isn't resolved. A new textbox object reverts to the default style
![image](https://github.com/user-attachments/assets/1af85088-cdbc-4119-8398-a59867b59e81)

**Comment by XDmitryK** [comment]
@Ephirion 
This feature is currently not supported. I have created a proposal 71984
